android/java/org/chromium/base/BraveCommandLineInitUtil.java

```diff
@@ -60,6 +62,21 @@ private static void appendBraveSwitchesAndArguments() {
             }
         }
 
+        File dayZeroExptFile = new File(TEST_DAY_ZERO_EXPT_FILE);
```

Comment:
@samartnik i have created a seperate option for day zero expt. i think it makes sense to combine all command line args from a file into one implementation. WDYT ? i can push a quick change if it's agreed upon

Reply:
it's just for testing, so either way is fine by me